### ISSUE TITLE
Native way to execute shell commands

### ISSUE THREAD

**Issue by y21** [jhallen/ivy#29]
Hello, is there a native way to execute shell commands like the function `system(const char*)` in C does (`stdlib.h`)?

If not, perhaps you could implement it and let it return stdout or -1 if an error ocurred/the exit code of the shell command.

**Comment by jallennk** [comment]
Yeah, this doesn't exist yet, and it's needed.  Also we need the equivalent of popen.
